heron/io/dlog/src/java/org/apache/heron/dlog/DLInputStream.java

```diff
@@ -100,6 +60,25 @@ private static LogRecordWithInputStream nextLogRecord(LogReader reader) throws I
     }
   }
 
+  public long getNumOfBytesRead() {
+    return numOfBytesRead;
+  }
+
+  /**
+   * Get input stream representing next entry in the
+   * ledger.
+   *
+   * @return input stream, or null if no more entries
+   */
+  private LogRecordWithInputStream nextLogRecord() throws IOException {
+    try {
+      return nextLogRecord(reader);
+    } catch (EndOfStreamException e) {
+      eos = true;
```

Comment:
Could it be helpful to have a log here?

Reply:
I didn’t really change this part of the file, it was my IDE automatically re-orgonized the code. But, I do think log here is helpful. Will update the pull request.